Sources/Lemmy-Swift-Client/Lemmy API/Requests/Login.swift

```diff
@@ -22,7 +22,9 @@ public struct LoginRequest: APIRequest {
 	}
 }
 public struct LoginResponse: APIResponse {
-	public let jwt: String
+    /// This is None in response to Register if email verification is enabled, or the server requires registration applications.
+    /// Or simply if the login is wrong
```

Comment:
This comment doesn't quite make sense.

What about something like this? @TheNoim does this capture the same meaning as what you were trying to say? Or did I misinterpret what you were trying to say?

```suggestion
	/// The JSON Web Token for the user.
	///
	/// This token can be passed into `auth` parameters of other requests to make authorized requests.
	///
	/// If you are registering a new account, this property will be `nil` if email verification is enabled, or if the server requires registration applications.
	///
	/// It will also be `nil` if the login is wrong.
```

Reply:
Sure, I just copied the doc string from the lemmy docs :D

Reply:
If I could find where the Lemmy docs are being generated, I'd make some updates and contribute some changes. But I can't find it.

Reference: https://eventfrontier.com/post/516